adsws/api/discoverer/utils.py

```diff
@@ -61,6 +63,10 @@ def bootstrap_local_module(service_uri, deploy_path, app):
         # ensure the current_app matches local_app and not API app
         view = local_app_context(local_app)(view)
 
+        if service_uri == 'adsws.solr_service.solr':
```

Comment:
what is the solr-service is ran in the remote mode? Maybe also need to add block in the remote_app_context?